front/app/modules/commercial/representativeness/admin/containers/Dashboard/index.tsx

```diff
@@ -5,28 +5,38 @@ import useFeatureFlag from 'hooks/useFeatureFlag';
 
 // components
 import { Box } from '@citizenlab/cl2-component-library';
-import { GraphsContainer } from 'components/admin/GraphWrappers';
 import Header from './Header';
 import ChartFilters from '../../components/ChartFilters';
+import EmptyState from './EmptyState';
+import ChartCards from './ChartCards';
+
+// tracks
+import { trackEventByName } from 'utils/analytics';
+import tracks from './tracks';
+
+const SHOW_EMPTY = true;
```

Comment:
In the future, this will be removed, and whether to show the empty state will instead be determined by whether the admin has uploaded reference (demographic) data